src/10-ipfs-0-4-0-released/index.md

```diff
@@ -0,0 +1,159 @@
+---
+date: 2016-03-17
+id: 10-ipfs-0-4-0-released
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+- {name: "10-ipfs-0-4-0-released", link: "./" }
+tags: modules
+title: IPFS 0.4.0 has been released
+author: Kyle Drake and Whyrusleeping
+collection: posts
+---
+
+[IPFS 0.4.0](https://github.com/ipfs/go-ipfs/releases/tag/v0.4.0) has been
+released! Among the many changes are a revamped implementation of the IPFS
+communication protocols, increased performance, improvements to IPNS (The
+Interplanetary Naming System), many bugfixes, and lots of new features to make
+IPFS even more powerful.
+
+## Why we're changing the protocol
+
+The most important change allows IPFS implementations to use pluggable
+stream multiplexers, such as [yamux](https://github.com/hashicorp/yamux),
+[spdystream](https://github.com/docker/spdystream), or
+[muxado](https://github.com/inconshreveable/muxado). Instead of locking IPFS
+permanently into a single multiplexer that won't work for every language or
+situation, this change allows each IPFS implementation to implement the multiplexers
+of their choosing, and then negotiate which to use during the initial connection
+handshake.
```

Comment:
What about linking to the `multistream` spec https://github.com/jbenet/multistream? So that the reader can understand what kind of black magic happens